lib/ParamedicSauceLabs.js

```diff
@@ -565,7 +565,7 @@ class ParamedicSauceLabs {
 
     getCommandForBuilding () {
         let cmd = this.config.getCli() + ' build ' + this.platformId + utilities.PARAMEDIC_COMMON_CLI_ARGS;
-        if(this.config.getArgs()){
+        if (this.config.getArgs()) {
```

Comment:
This change should be separate from this PR. Wouldn't even need a review or PR itself IMHO.

Reply:
Done: https://github.com/apache/cordova-paramedic/commit/c575f2de8075e272444f2d2bcd3dccd2258de60d

Reply:
And rebased so it is gone here.